### PR TITLE
Fix ProductImageForm

### LINKED ISSUE #1260: Problems with product image view in the dashboard
There are few problems with the view for editing images in the dashboard:
- colliding labels at the top ("Point of interest")
- checkbox "clear" is not working
- button to select a new image is not working

image

### PR DESCRIPTION
**Pull request by koradon**
Change link to image in ProductImageForm.
Instead of full path to the image display only image file name.
before:
screenshot from 2017-11-07 15-49-54

after:
screenshot from 2017-11-07 15-49-30

Fixes #1260

### CODE REVIEW

**Comment by patrys** [comment]
Please provide at least a brief summary of the changes when submitting a pull request. Ideally along with before/after screenshots if it affects the UI. 👍

**Comment by koradon** [comment]
I am adding it right now :)

**Comment by patrys** [comment]
It's kind of weird, "pick the most relevant..." refers to the image widget above but there's a file selection field between those two. All other fields have a label (stylized as one) before the actual input.

**Comment by koradon** [comment]
@patrys Yea i know, this widget needs proper frontend makeover.
I have just changed what i could from the backend side.

File: saleor/dashboard/templatetags/utils.py [hunk 1]
```diff
@@ -27,9 +27,15 @@ def construct_get_query(context, **params):
 
 @register.filter
 def is_versatile_image_ppoi_click_widget(field):
+    print(type(field.field.widget), isinstance(field.field.widget, VersatileImagePPOIClickWidget))
```
Comment (elwoodxblues) [inline saleor/dashboard/templatetags/utils.py]: Debug code left

File: saleor/dashboard/templatetags/utils.py [hunk 2]
```diff
@@ -27,9 +27,15 @@ def construct_get_query(context, **params):
...
     return isinstance(field.field.widget, VersatileImagePPOIClickWidget)
 
 
+@register.filter
+def is_image_preview_widget(field):
+    return isinstance(field.field.widget, ImagePreviewWidget)
```
Comment (elwoodxblues) [inline saleor/dashboard/templatetags/utils.py]: We could as well add a docstring here.

File: saleor/dashboard/templatetags/utils.py (hunk 1 above)
Comment (elwoodxblues) [reply saleor/dashboard/templatetags/utils.py]: Could we add a docstring to this function explaining in what cases we expect the field to be an instance of `VersatileImagePPOIClickWidget `? This logic is a bit confusing and we ourselves had problems with figuring out in what cases which of the two image widgets is used.

**Comment by koradon** [comment]
New look by @dominik-zeglen 
screenshot from 2017-11-08 12-23-42

**Comment by patrys** [comment]
Nitpicking: "Primary Point of Interest" is using title case where other labels use sentence case ("Short description").

**Comment by patrys** [comment]
Honest question: does linking to the current image solve an actual problem?

**Comment by elwoodxblues** [comment]
I think it doesn't have to be a link, could be just a text. Also, shouldn't we display the file name inside the file input?

**Comment by patrys** [comment]
@elwoodxblues I'm asking since it's hard for me to imagine why I would be interested in the file's name (it's usually gibberish) when I have the preview displayed right under that. The customers are never directly exposed to the name so it's mostly an implementation detail.

**Pull request merged**